sdk/authorization/arm-authorization/src/authorizationManagementClientContext.ts

```diff
@@ -13,7 +13,7 @@ import * as msRest from "@azure/ms-rest-js";
 import * as msRestAzure from "@azure/ms-rest-azure-js";
 
 const packageName = "@azure/arm-authorization";
-const packageVersion = "0.1.0";
+const packageVersion = "8.3.1";
```

Comment:
shouldnt this be 8.3.2?

Reply:
yup it should match the version in package.json

Reply:
Done and Resolving this